exercises/28_showmem/Makefile

```diff
@@ -3,43 +3,12 @@
 ## showmem makefile
 ##
 
-SRC	=	showmem.c
-SIDE	=	../01_putchar/putchar.c ../03_put_nbr/put_nbr.c ../05_putstr/putstr.c ../06_strlen/strlen.c
-SIDE += ../assert_match_stdout.c
-
-OBJ	=	$(SRC:.c=.o) $(SIDE:.c=.o)
-COV	=	./*.gcda ./*.gcno
-
-NAME	=	showmem.bin
-TESTNAME=	test.bin
-TESTFILE=	tests.c
-CRITOPATH=	../../criterion/
-
-all:	$(NAME)
-
-$(NAME):	$(OBJ)
-	@gcc -o $(NAME) $(SRC) $(SIDE) -g3
-	@echo "$(NAME) compiled successfully"
-
-clean:
-	@rm -f $(OBJ)
-	@echo "Cleaned .o"
-
-fclean: clean
-	@rm -f $(NAME)
-	@echo "Cleaned '$(NAME)'"
-	@rm -f $(TESTNAME) $(COV)
-	@echo "Cleaned '$(TESTNAME)' binary & residual files"
-
-re:	fclean all
-
-test:
-	@gcc -fprofile-arcs -ftest-coverage -w -o $(TESTNAME) $(TESTFILE) $(SRC) $(SIDE) \
-		-g3 -I$(CRITOPATH)include -L$(CRITOPATH)lib -lcriterion -lm
-
-run:	fclean test
-	@./$(TESTNAME) --verbose
-runc:	run
-	@gcovr --exclude tests.c --exclude-unreachable-branches --sort-uncovered --print-summary
-
-.PHONY: all clean fclean re test run runc
+SRC	:= showmem.c
+SRC += ../01_putchar/putchar.c

```

Comment:
wouldn't it be better to group added filenames together with using backslash as a return marker? chained += is unnecessary here